dataone-utils/src/main/java/org/irods/jargon/dataone/configuration/RestConfiguration.java

```diff
@@ -246,4 +255,43 @@ public void setAuthType(String authType) {
 		this.authType = authType;
 	}
 
+	/**
+	 * Gets the authorization type used by the member node service, which determines how the member
+	 * node service checks for authorization in endpoints that require it. The currently available
+	 * types are {@code http-header} and {@code always-allow}.
+	 *
+	 * @return the authorization type.
+	 */
+	public String getMemberNodeAuthType() {
+		return memberNodeAuthType;
+	}
+
+	/**
+	 * Sets the authorization type used by the member node service, which determines how the member
+	 * node service check for authorization in endpoints that require it. The currently available
+	 * types are {@code http-header} and {@code always-allow}.
+	 *
+	 * @param memberNodeAuthType the authorization type.
+	 */
+	public void setMemberNodeAuthType(String memberNodeAuthType) {
+		this.memberNodeAuthType = memberNodeAuthType;
+	}
+
+	/**
+	 * Obtains an authorization checker to use for the member node service based on the value of the
+	 * value of {@code memberNodeAuthType}. If the value is equal to "http-header" (case insensitive),
+	 * then an instance of {@link HttpHeaderAuthChecker} is returned. Otherwise, and instance of
+	 * {@link AlwaysAllowAuthChecker} is returned.
+	 *
+	 * @return an instance of {@link AuthChecker}.
+	 */
+	public AuthChecker getAuthChecker() {
+		String authType = getMemberNodeAuthType();
+		logger.error("getting the auth checker for auth type: {}", authType);
+		if ("http-header".equalsIgnoreCase(authType)) {
+			return new HttpHeaderAuthChecker();
+		} else {
+			return new AlwaysAllowAuthChecker();
+		}
+	}
```

Comment:
This is the code determines how the member node service checks for authentication information.